apps/examples/connect_test/connect_test.c

```diff
@@ -0,0 +1,358 @@
+	opt->hostname = argv[3];
+	return 0;		
+}
+
+int _ct_parse_server(struct ct_options *opt,int argc, char *argv[])
+{
+ 	if (argc < 9 || argc > 14) {
+		return -1;
+	}
+	if (_parse_security_str(opt, argv[3], argv[4]) != 0) {
+		return -1;
+	}
+	opt->ssid = argv[2];
+	opt->softap_ssid = argv[5];
+	opt->softap_password = argv[6];
+	opt->softap_channel = atoi(argv[7]);
+	opt->repeat = atoi(argv[8]);
+
+	int data_idx = 9;
+	if (argc > data_idx && (!strncmp(argv[data_idx], "udp", 4))) {
+		opt->is_dtls = 0;
+		opt->dtls_data_size = atoi(argv[++data_idx]);
+		data_idx++;
+	} else if (argc > data_idx && !strncmp(argv[data_idx], "dtls", 5)) {
+		opt->is_dtls = 1;
+		opt->dtls_data_size = atoi(argv[++data_idx]);
+		data_idx++;
+	}  else {
+		opt->is_dtls = 0;
+		opt->dtls_data_size = 64;
+	}
+
+	if (argc > data_idx && !strncmp(argv[data_idx], "tcp", 4)) {
+		opt->is_tls = 0;
+		opt->tls_data_size = atoi(argv[++data_idx]);
+		data_idx++;
+	} else if (argc > data_idx && !strncmp(argv[data_idx], "tls", 4)) {
+		opt->is_tls = 1;
+		opt->tls_data_size = atoi(argv[++data_idx]);
+		data_idx++;
+	} else {
+		opt->is_tls = 0;
+		opt->tls_data_size = 64;
+	}
+
+	int hostname_idx = data_idx;
+	if (argc > hostname_idx) {
+		opt->path = argv[hostname_idx];
+	}
+	return 0;
+}
+
+int _ct_parse_client(struct ct_options *opt, int argc, char *argv[])
+{
+	if (argc < 9 || argc > 13) {
+		return -1;
+	}
+	if (_parse_security_str(opt, argv[3], argv[4]) != 0) {
+		return -1;
+	}
+	opt->ssid = argv[2];
+	opt->softap_ssid = argv[5];
+	opt->softap_password = argv[6];
+	opt->softap_channel = atoi(argv[7]);
+	opt->repeat = atoi(argv[8]);
+
+	int data_idx = 9;
+	if (argc > data_idx && (!strncmp(argv[data_idx], "udp", 4))) {
+		opt->is_dtls = 0;
+		opt->dtls_data_size = atoi(argv[++data_idx]);
+		data_idx++;
+	} else if (argc > data_idx && !strncmp(argv[data_idx], "dtls", 5)) {
+		opt->is_dtls = 1;
+		opt->dtls_data_size = atoi(argv[++data_idx]);
+		data_idx++;
+	}  else {
+		opt->is_dtls = 0;
+		opt->dtls_data_size = 64;
+	}
+
+	if (argc > data_idx && !strncmp(argv[data_idx], "tcp", 4)) {
+		opt->is_tls = 0;
+		opt->tls_data_size = atoi(argv[++data_idx]);
+		data_idx++;
+	} else if (argc > data_idx && !strncmp(argv[data_idx], "tls", 4)) {
+		opt->is_tls = 1;
+		opt->tls_data_size = atoi(argv[++data_idx]);
+		data_idx++;
+	} else {
+		opt->is_tls = 0;
+		opt->tls_data_size = 64;
+	}
+	return 0;
+}
+
+int _ct_parse_commands(struct ct_options *opt, int argc, char *argv[])
+{
+	int ret = 0;
+	ct_type_e options = _ct_get_opt(argc, argv);
+	if (options == CT_TYPE_ERR) {
+		return -1;
+	}
+
+	opt->func = g_function_table[options];
+	if (opt->func == NULL) {
+		return -1;
+	}
+
+	if (g_function_parser_table[options]) {
+		ret = (g_function_parser_table[options])(opt, argc, argv);
+	}
+	return ret;
+}
+
+
+void _ct_process(int argc, char *argv[])
+{
+	struct ct_options opt;
+	memset(&opt, 0, sizeof(struct ct_options));
+	opt.repeat = 1;
+	opt.softap_channel = 1;
+#ifdef __LINUX__
```

Comment:
I think this test doesn't support linux. please remove this.

Reply:
Changes Done as suggested